.circleci/config.yml

```diff
@@ -29,6 +29,48 @@ fetch_data: &fetch_data
       ./scripts/fetch_kaggle_dataset.sh
 
 jobs:
+  section_07_test_app:
```

Comment:
@solegalli new jobs added:
- prefaced with `section_07`
- exclusively run with tox for consistency

Reply:
good idea this prefacing thing